tests/testthat/test-skim_print.R

```diff
@@ -31,23 +34,23 @@ test_that("knit_print produced expected results", {
   expect_is(input, "knit_asis")
   expect_length(input, 1)
 
-  expect_match(input, "**Skim summary statistics**", fixed = TRUE)
-  expect_match(input, "<table style='width: auto;'")
```

Comment:
If I remember right, there was a reason to produce the table here.  I think this has to do with the behavior of knit_asis. Can you knit a simple example into a doc and confirm that the summary data appears in an html table?

Reply:
It actually does not appear as a table, and it doesn't make sense as a table since it is not tabular data, so it would not pass a standards test (and was a mess before).  For HTML if anything it should be a list, ideally a DL but RMarkdown doesn't support DL.    If we made it a table we would have to make the text into the row labels.

Reply:
You also have to be aware of how that gets passed to pdf .... one of the issue in the knitted html also was that there were two tables and the second one was floating up to the right, so if we want to really do tables we would have to add CSS to make sure that there is a clear after the first table.  OR we could try to figure out how to make it work as a single table if we can make that also work with the pdf.

Reply:
Actually annoyingly enough it is putting it into a table with one cell.